code/server/routes/api.js

```diff
@@ -43,6 +44,24 @@ router.get('/randomUser', function(req, res){
 	});
 });
 
+// Getting a specific user
+// Specify username in GET variable
+router.get('/getUser', function(req, res) {
+
+    if (req.query.user) {
+        User.getUser(req.query.user).then(function(user) {
+            if (user) {
+              delete user.dataValues.password // probably not the best idea to send this over the wire
```

Comment:
I'm a bit out of practise with javascript, should this have a semicolon or is it intentionally not there?